src/util-time.c

```diff
@@ -138,6 +138,20 @@ void CreateIsoTimeString (const struct timeval *ts, char *str, size_t size)
     }
 }
 
+void TimeToIsoTimeString (time_t time, char *str, size_t size)
```

Comment:
The output is the same, but without microseconds. It probably makes more sense to convert from time_t to timeval and use CreateIsoTimeString instead. What do you think?